### PR TITLE
5273 - edit other stats; delete modal

### PR DESCRIPTION
**Pull request by codyseibert**
- new view for edit
- new sequences + actions for delete / edit

Screen Shot 2020-05-27 at 5 21 17 PM

### CODE REVIEW

File: web-client/src/presenter/actions/deleteOtherStatisticsAction.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+/**
+ * clears the other statistics off the case
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.get the cerebral get function
+ * @param {object} providers.applicationContext the applicationContext
+ * @param {object} providers.path the next object in the path
+ * @returns {Promise<*>} the success or error path
+ */
+export const deleteOtherStatisticsAction = async ({
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/deleteOtherStatisticsAction.js:11]: Tests for these new actions please?

File: web-client/src/presenter/actions/setEditOtherFormAction.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { state } from 'cerebral';
+
+/**
+ * sets the state.form using the damages and litigationCosts from the caseDetail
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.get the cerebral get function
+ * @param {object} providers.store the cerebral store used for setting the state.caseDetail
+ */
+export const setEditOtherFormAction = ({ get, store }) => {
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/setEditOtherFormAction.js]: Tests - might also want to name this `setEditOtherStatisticsFormAction` for clarity

File: web-client/src/presenter/actions/submitEditOtherStatisticsAction.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+/**
+ * submits the edit other statistics form
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.get the cerebral get function
+ * @param {object} providers.applicationContext the applicationContext
+ * @param {object} providers.path the next object in the path
+ * @returns {Promise<*>} the success or error path
+ */
+export const submitEditOtherStatisticsAction = async ({
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/submitEditOtherStatisticsAction.js:11]: Tests

File: web-client/src/views/CaseDetail/EditDeficiencyStatistics.jsx [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+    submitAddDeficiencyStatisticsSequence:
+      sequences.submitAddDeficiencyStatisticsSequence,
+    updateAddDeficiencyFormValueSequence:
+      sequences.updateAddDeficiencyFormValueSequence,
+    updateFormValueSequence: sequences.updateFormValueSequence,
+    validateAddDeficiencyStatisticsSequence:
+      sequences.validateAddDeficiencyStatisticsSequence,
+    validationErrors: state.validationErrors,
+  },
+  function StatisticsForm({
```
Comment (rachaelparris) [inline web-client/src/views/CaseDetail/EditDeficiencyStatistics.jsx]: Accidental commit of this file? If not, this name should be `EditDeficiencyStatistics`

File: web-client/src/presenter/sequences/deleteOtherStatisticsSequence.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+import { setAlertSuccessAction } from '../actions/setAlertSuccessAction';
+import { setCaseDetailPageTabFrozenAction } from '../actions/CaseDetail/setCaseDetailPageTabFrozenAction';
+import { setSaveAlertsForNavigationAction } from '../actions/setSaveAlertsForNavigationAction';
+import { showProgressSequenceDecorator } from '../utilities/sequenceHelpers';
+
+export const deleteOtherStatisticsSequence = [
+  showProgressSequenceDecorator([
+    deleteOtherStatisticsAction,
+    {
+      error: [],
```
Comment (kkoskelin) [inline web-client/src/presenter/sequences/deleteOtherStatisticsSequence.js:14]: I think you need to acquire the errors from props?

File: web-client/src/presenter/actions/setEditOtherStatisticsFromAction.test.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import { runAction } from 'cerebral/test';
+import { setEditOtherStatisticsFormAction } from './setEditOtherStatisticsFormAction';
+
+describe('setEditOtherStatisticsFormAction', () => {
+  it('adds the props.field key to state.fieldOrder', async () => {
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/setEditOtherStatisticsFromAction.test.js]: Test description needs to be updated

**Review by rachaelparris — APPROVED** [review]
One test description just needs an update

**Pull request merged**